multinode/bootstrapping.md

```diff
@@ -0,0 +1,189 @@
+# Data Node Bootstrapping [](distdb-bootstrap)
+
+Scaling up TimescaleDB is achieved by adding data nodes to an access
+node using the [`add_data_node`][add_data_node] and
+[`attach_data_node`][attach_data_node] functions. In order for a data
+node to be available to the access node it is necessary to ensure that
+it is properly setup to work as a data node. This process is referred
+to as *bootstrapping the data node*.
+
+## Authentication during bootstrapping
+
+All connections to the data node are made as the current user on the
+access node. Which requires that:
+
+1. The current user exists on both the data node and access node.
+2. The data node is set up to handle logins using either certificates
+   or password files, as described in the [Data Node Authentication
+   Chapter][distdb-auth].
+
+## Adding data nodes to the access node
+
+A new data node is added to the access node using the
+[`add_data_node`][add_data_node] command.
+
+```
+SELECT * FROM add_data_node('data_node_1', 'data1.example.com');
+```
+
+The access node has to ensure that the database on the data node is
+properly configured by either bootstrapping the database or verifying
+that the database exists and is correctly configured. This is
+controlled with the `bootstrap` option, which can be either `true`
+(indicating that the data node should be bootstrapped) or `false`
+(indicating that the data node will only be validated, but no
+bootstrapping will take place).
+
+```
+SELECT * FROM add_data_node('data_node_1', 'data1.example.com',
+                            boostrap => false);
+```
+
+The default value for the `bootstrap` option is `true`, which means
+that the data node is bootstrapped when added.
+
+The steps to boostrap the data node are:
+
+1. Ensure that there is a database on the data node.
+2. Ensure that the Timescale extension is available in the database.
+3. Set metadata on the data node.
+
+## Setting up the database on the data node
+
+The database to bootstrap on the data node is given by the name of the
+current database on the access node. For example, if you are connected
+to the `postgres` database on the access node, the database name will
+be `postgres` on the data node as well.
+
+To provision the data node in a different database on the instance use
+the `database` option to the [`add_data_node`][add_data_node] command,
+for example:
+
+```
+SELECT * FROM add_data_node('data_node_1', 'data1.example.com',
+                            database => 'my_database');
+```
+
+This is particularly useful if you want to experiment with having data
+nodes and access nodes in the same Postgres instance.
+
+### Bootstrapping the database
+
+If `bootstrap` is true (which is the default), the access node will
+first connect to either the `postgres` database or, if the `postgres`
+database does not exist, to the `template0` database on the data node,
+and create a new database:
+
+```
+CREATE DATABASE <database>
```

Comment:
```suggestion
CREATE DATABASE < database >
```
This is necessary due to a quirk of our markdown parser that treats `<database>` as a potential React JSX element, even within code blocks.  Alternatively, you could use symbols other than `<>`

Reply:
What is used elsewhere? I have no strong opinion on what to use, but this is intended to be a [replacable](https://tdg.docbook.org/tdg/4.5/replaceable.html)

Reply:
Switched to use `<i>` tags, not sure if that is supported, but I see it in other parts of the document.